src/main/java/frc/robot/subsystems/swerve/module/Module.java

```diff
@@ -104,14 +107,15 @@ public void updateInputs() {
 
 		fixDriveInputsCoupling();
 
-		driveInputs.velocityMetersPerSecond = toDriveMeters(driveInputs.uncoupledVelocityPerSecond);
-		driveInputs.positionsMeters = Arrays.stream(driveInputs.uncoupledPositions).mapToDouble(this::toDriveMeters).toArray();
+		driveInputs.velocityMetersPerSecond = toDriveMeters(couplingInputs.uncoupledVelocityPerSecond);
+		driveInputs.positionsMeters = Arrays.stream(couplingInputs.uncoupledPositions).mapToDouble(this::toDriveMeters).toArray();
 
 		moduleInputs.isClosedLoop = isClosedLoop;
 		moduleInputs.targetState = targetState;
 
 		Logger.processInputs(constants.logPath(), moduleInputs);
 		Logger.processInputs(driveStuff.logPath(), driveInputs);
+		Logger.processInputs(driveStuff.logPath(), couplingInputs);
```

Comment:
if you're splitting it in code split it also in log (do different log path)

Reply:
i dont agree with that, i prefer the drive inputs to be toghether